File: Version_PHP/CodigoJAVA/delito.java
package Version_PHP.CodigoJAVA;

public class delito extends direccion {

    private String Calle;
    private String TipoDelito;
    private String Descripcion;

    // constructor implementacion de concepto de composicion para que al momento de
    // borrar el delito tambien se borre su direccion donde ocurrio
    public delito(String departamneto, String municipio, String colonia, String bloque, String Calle, String TipoDelito,
            String Descripcion) {
        super(departamneto, municipio, colonia, bloque);
        this.Calle = Calle;
        this.TipoDelito = TipoDelito;
        this.Descripcion = Descripcion;
        direccion objdireccionDelito = new direccion(departamneto, municipio, colonia, bloque);
    }

    // geters
    public String getCalle() {
        return Calle;
    }

    public String getTipoDelito() {
        return TipoDelito;
    }

    public String getDescripcion() {
        return Descripcion;
    }

    // setters
    public void setCalle(String calle) {
        Calle = calle;
    }

    public void setTipoDelito(String tipoDelito) {
        TipoDelito = tipoDelito;
    }

    public void setDescripcion(String descripcion) {
        Descripcion = descripcion;
    }

}

File: CodigoDelito/VersionWEB/php/Delito.php
<?php

include_once("Direccion.php");
class Delito extends Direccion
{
    private $calle;
    private $TipoDelito;
    private $Descripcion;
    protected $property;

    

    // constructor implementacion de concepto de composicion para que al momento de
    // borrar el delito tambien se borre su direccion donde ocurrio
    public function __construct($departamento, $municipio, $colonia, $numeroBloque, $calle, $TipoDelito, $Descripcion)
    {
        $this->calle = $calle;
        $this->TipoDelito = $TipoDelito;
        $this->Descripcion = $Descripcion;

        $objDireccionDelito = new Direccion($departamento, $municipio, $colonia, $numeroBloque, $calle);
    }

    //getters

    public function getCalle(){
        return $this->calle;
    }
    public function getTipoDelito(){
        return $this->TipoDelito;
    }
    public function getDescripcion(){
        return $this->Descripcion;
    }
    

    
    //setters
    public function setCalle($calle){
        $this->calle = $calle;
        return $this;
    }
    public function setTipoDelito($TipoDelito){
        $this->TipoDelito = $TipoDelito;
        return $this;
    }
    public function setDescripcion($Descripcion){
        $this->Descripcion = $Descripcion;
        return $this;
    }

}



?>